Apps/phnetskope/netskope.json

```diff
@@ -8,13 +8,13 @@
     "logo_dark": "logo_netskope_dark.svg",
     "product_name": "Netskope",
     "product_version_regex": ".*",
-    "publisher": "Phantom",
-    "license": "Copyright (c) 2018-2019 Splunk Inc.",
-    "app_version": "1.0.4",
+    "publisher": "Netskope",
+    "license": "Copyright (c) 2018-2020 Splunk Inc.",
+    "app_version": "2.0.0",
```

Comment:
latest_tested_versions?

Reply:
@phantom-james It is a community app, and we do not have an asset to test the app.